landscape/README.md

```diff
@@ -0,0 +1,36 @@
+## Goal
+
+The [SAFE roadmap](../roadmap.md) includes describing the landscape of
+cloud-native security. We evaluated categories in the CNCF Landscape and
```

Comment:
It would be good to link CNCF Landscape in its first appearance in the doc (ie here)